doc/src/installation.md

```diff
@@ -1,319 +1,199 @@
 # Installation
 
-All the instructions below work on Linux, macOS and Windows.
+Follow the steps below to install and run LPython on Linux, Windows or macOS.
+
+## Prerequisites
+- ### Install Conda
+  Follow the instructions provided [here](https://github.com/conda-forge/miniforge/#download) to install Conda on your platform (Linux, macOS and Windows) using a conda-forge distribution called Miniforge.
+  
+  For Windows, these are additional requirements:
+  - Miniforge Prompt
+  - Visual Studio (with "Desktop Development with C++" workload)
+
+- ### Set up your system
+    - Linux
+        - Make sure you have `g++` and `cmake` installed. If not, install them using the below command:
+
+            ```bash
+            sudo apt-get install g++ cmake
+            ```
+
+        - Run the following command to install the build dependencies:
+
+            ```bash
+            sudo apt-get install binutils-dev build-essential zlib1g-dev bison re2c flex
```

Comment:
Also, I think explicitly installing flex might not be needed on linux.

Reply:
An explicit installation of `flex` is required for downloading other header files required for build.

Reply:
I think `flex` should already be installed in the `conda` environment. I guess you missed to create a conda environment?